timm/models/mobilenetv3.py

```diff
@@ -7,18 +7,19 @@
 Hacked together by / Copyright 2019, Ross Wightman
 """
 from functools import partial
-from typing import List
+from typing import Callable, List, Optional, Tuple
 
 import torch
 import torch.nn as nn
 import torch.nn.functional as F
+from torch import Tensor
```

Comment:
I'd prefer to keep things as torch.Tensor ... a few more chars but already quite common through the code

Reply:
Got it, will change.